components/NativeShellWebView.js

```
@@ -65,130 +62,131 @@ window.onerror = console.error;
+					BackHandler.exitApp();
+					break;
+				case 'enableFullscreen':
+					rootStore.set({ isFullscreen: true });
+					break;
+				case 'disableFullscreen':
+					rootStore.set({ isFullscreen: false });
+					break;
+				case 'downloadFile':
+					console.log('Download item', data);
+					/* eslint-disable no-case-declarations */
+					const url = new URL(data.item.url);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)